libs/dal/src/repositories/user/user.repository.ts

```diff
@@ -88,4 +88,8 @@ export class UserRepository implements IUserRepository {
   bulkWrite(bulkOperations: any, ordered: boolean): Promise<any> {
     return this.userRepository.bulkWrite(bulkOperations, ordered);
   }
+
+  estimatedDocumentCount(): Promise<number> {
+    return this.userRepository.estimatedDocumentCount();
+  }
```

Comment:
We already have the BaseRepository.estimatedDocumentCount(), i think we could use it instead of overriding it on child classes. what do you think about it?

Reply:
Yes, that actually make sense. I added this as place holder if I want to make change in future also to bring parity. Surely using directly from BaseRepository is a good. Let me change that in next iteration.